multihost_test/Bugzillas/test_Regression.py

```diff
@@ -538,3 +538,249 @@ def test_bz1114081_1285547(self, multihost, create_backup):
         assert "Never logged in" not in last
         client.run_command(f"lastlog --clear --user {user}")
         client.run_command(f"userdel -rf {user}")
+
+    @pytest.mark.tier1
+    def test_bz1498628(self, multihost):
+        """
+        :title:shadow-utils: Update to get newuidmap and newgidmap binaries
+        :id: 3af5735a-fd87-11ee-8c23-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1498628
+        :steps:
+          1. Add a user and set password for user.
+          2. Retrieve the UID (User ID) and GID (Group ID) of the created user.
+          3. Checks for any sub-UID and sub-GID entries for the user "testUser1" in
+            the /etc/subuid and /etc/subgid files respectively.
+          4. Sanity check for binaries attributes
+          5. Test that newuidmap and newgidmap have manual pages
+          6. Test that newuidmap and newgidmap works without altering the capability bounding set
+          7. Test that newuidmap and newgidmap does not allow mapping subuids outside of allowed range
+          8. Test that newuidmap and newgidmap works when called from process
+            with cap_sys_admin removed from bounding set
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+          5. Should succeed
+          6. Should succeed
+          7. Should succeed
+          8. Should succeed
+        """
+        client = multihost.client[0]
+        pidfile = "/tmp/pidfile.txt"
+        client.run_command("useradd testUser1")
+        password = "Secret123"
+        passwd_cmd = f'passwd --stdin testUser1'
+        multihost.client[0].run_command(passwd_cmd, stdin_text=password, raiseonerr=False)
+        client.run_command("grep testUser1 /etc/subuid")
+        subuid = client.run_command("grep testUser1 /etc/subuid | cut -f2 -d':'").stdout_text.split()[0]
+        testUserUid = client.run_command("id -u testUser1").stdout_text.split()[0]
+        testUserGid = client.run_command("id -g testUser1").stdout_text.split()[0]
```

Comment:
Move this part of the code before looking at the subuids so that it matches the steps

Reply:
One more line please